test/test_tree_2d_mhd.jl

```diff
@@ -183,7 +183,7 @@ end
     end
 end
 
-@trixi_testset "elixir_mhd_orszag_tang.jl with flux_hll" begin
+@trixi_testset "elixir_mhd_orszag_tang.jl with FluxHLL(min_max_speed_naive)" begin
```

Comment:
I'm not sure whether we really want to test the naive version here or the Davis version.